### ISSUE TITLE
Doc Typo:  matplotlib.axes.Axes.get_yticklabels  / Axis.get_ticklabels

### ISSUE THREAD

**Issue by jason-neal** [matplotlib/matplotlib#11034]
### Bug report

**Bug summary**
[get_yticklabels ](https://matplotlib.org/api/_as_gen/matplotlib.axes.Axes.get_yticklabels.html#matplotlib.axes.Axes.get_yticklabels) doc refers to x ticks. 

```
Axes.get_yticklabels(minor=False, which=None)
    Get the x tick labels as a list of Text instances.
```

I went to make a PR but was unsure where to make the change.

I found a get_yticklabels() function in `axes/_base.py` but it has correct docs.
https://github.com/matplotlib/matplotlib/blob/90ac52366535e71827738feb3dea15317d233a96/lib/matplotlib/axes/_base.py#L3526-L3528  

It does however use a  `get_ticklabels` method.
Finding a `get_ticklabels` method of the `Axis` class I see that the documentation specifically references x ticks. 
https://github.com/matplotlib/matplotlib/blob/7d7692f966e89ec8060e420dceb25154d9480f49/lib/matplotlib/axis.py#L1246-L1248

This may be where the typo is coming from. But I am not sure if this is just a different typo.

**Expected outcome**
Should be probably be
```
Axes.get_yticklabels(minor=False, which=None)
    Get the y tick labels as a list of Text instances.
```

Some guidance on where to make the changes and I can do a PR.
Thanks

**Comment by jklymak** [comment]
Someone already fixed the first typo.  The second type (in `axis.py`) could probably be fixed.

**Comment by anntzer** [comment]
Closed by #11043.
